Lib/importlib/_bootstrap_external.py

```diff
@@ -1367,7 +1367,11 @@ def _find_parent_path_names(self):
 
     def _get_parent_path(self):
         parent_module_name, path_attr_name = self._find_parent_path_names()
-        return getattr(sys.modules[parent_module_name], path_attr_name)
+        try:
+            module = sys.modules[parent_module_name]
+        except KeyError:
+            return sys.path
```

Comment:
This should raise `ModuleNotFoundError` as trying to import w/o the parent module already having been imported is an error.